### DISCUSSION TITLE
Map Display Issues

### DISCUSSION THREAD

**Discussion by elyna-grapstein** [earthlab-education/Earth-Analytics-AY24#279] [General]
Is anyone else struggling to get their map to display on their portfolio page? It seems like it should be working on my end, but doesn't seem to load. Curious what I could look for to troubleshoot if the code seems to have been running fine and downloads are placed in correct locations.

**Comment by BioNCphilic** [comment]
I had to try a few times and then realized that it takes a while to load and become functional. The line <embed type="text/html" src="uttc.html" width="600" height="600"> needs to be edited to read <embed type="text/html" src="img/uttc.html" width="600" height="600">. The main thing is to be patient if all that is correct.
